remove console.warn from local authentication (#19803)
expo/expo: packages/expo-local-authentication/CHANGELOG.md

### 🐛 Bug fixes

- Added estimate of supported authentication types for Android. ([#8431](https://github.com/expo/expo/pull/8431) by [@bycedric](https://github.com/bycedric))

### 💡 Others

> - Remove console.warn from LocalAuthentication, preventing to run it every time that user cancel authentication or authentication fail. ([#19803](https://github.com/expo/expo/pull/19803) by [@GustavoMont](https://github.com/GustavoMont))

Reviewer: ```suggestion
- Removed `console.warn` from the `authenticateAsync` function, preventing running it every time the user cancels authentication or authentication fails. ([#19803](https://github.com/expo/expo/pull/19803) by [@GustavoMont](https://github.com/GustavoMont))
```
